utils/docker/Dockerfile

```diff
@@ -17,9 +17,11 @@ RUN apt-get -q update && \
     sqlite3 \
     python-pip \
     libsqlite3-dev \
-    libncurses5-dev
+    libncurses5-dev \
+    vim
```

Comment:
seems unfair to arbitrarily choose editors to include in the image; if you want VIM you should install it manually

Reply:
I think it's fine to include vim as its bundled with major linux distributions anyway. Besides, I don't see it as a major issue.

Reply:
I'd really prefer this not be here. There's no justification for it at all. Docker images are supposed to be kept as small as possible to do their job.

Reply:
I agree Docker images should be small but thats when deploying in production. This is really for just development convenience. I don't think it's unreasonable given our use case. Anyway, we can leave it out for now and add some way to customize such addons.